addon/templates/components/polaris-spinner.hbs

```diff
@@ -1,8 +1,10 @@
-{{svg-jar
-  spinnerSource
-  data-test-spinner=true
+{{!-- TODO: replace this with polaris-image when implemented. --}}
+<img
+  alt=""
+  src={{spinnerSVG}}
+  class={{spinnerClass}}
+  draggable="false"
   role="status"
-  id=svgElementId
-  class=spinnerClass
-  aria-label=accessibilityLabel
-}}
+  aria-label={{accessibilityLabel}}
+  data-test-polaris-spinner
+>
```

Comment:
Does this change mean we can remove the `SvgHandling` mixin from `polaris-spinner`?

Reply:
yasss, let's remove that :D